Replace ListView by RecyclerView.

Rajnish23 opened the pull request:
Fixed - If an item is expanded random list items getting expanded.

## Description

Replace the Listview with RecyclerView.
Fixes - In CityHistoryAdapter if any Text is expanded randomly few items were getting expanded.

Fixes #(issue)

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

Swati4star commented on Android/app/gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -0,0 +1,6 @@
+#Thu Oct 01 11:48:10 AST 2020
```
revert this file

Swati4star commented on Android/app/gradlew:
```diff
@@ -0,0 +1,172 @@
+#!/usr/bin/env sh
```
remove this file

Swati4star commented on Android/app/gradlew.bat:
```diff
@@ -0,0 +1,84 @@
+@if "%DEBUG%" == "" @echo off
```
remove this file too

Please add required changes only

Rajnish23 commented:
Sure 👍

On Fri, 2 Oct 2020, 1:36 pm Swati Garg, <notifications@github.com> wrote:

> *@Swati4star* requested changes on this pull request.
> ------------------------------
>
> In Android/app/gradle/wrapper/gradle-wrapper.properties
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#discussion_r498744098>
> :
>
> > @@ -0,0 +1,6 @@
> +#Thu Oct 01 11:48:10 AST 2020
>
> revert this file
> ------------------------------
>
> In Android/app/gradlew
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#discussion_r498744141>
> :
>
> > @@ -0,0 +1,172 @@
> +#!/usr/bin/env sh
>
> remove this file
> ------------------------------
>
> In Android/app/gradlew.bat
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#discussion_r498744325>
> :
>
> > @@ -0,0 +1,84 @@
> +@if "%DEBUG%" == "" @echo off
>
> remove this file too
>
> Please add required changes only
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#pullrequestreview-501027577>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHUZ7SPH4L4PDA5SOP7XC4DSIWUJBANCNFSM4SA45FIA>
> .
>

Rajnish23 reviewed (commented):
Done with Required Changes.

Swati4star commented:
There are still many gradle files which need to be reverted. Please see the "files changed" by this PR

Rajnish23 commented:
> There are still many gradle files which need to be reverted. Please see the "files changed" by this PR

There are total 7 files which are changed by this Pull Request. 4 of  them are CityHistoryActivity, CityHistoryAdapter, CityHistoryListItem and activity_city_history.xml. I have deleted the gradlew file and reverted the changes made in gradle-wrapper.properties. And the last one which is gradle-wrapper.jar.
I didn't made any changes to all other files in the project..

Swati4star commented:
The gradle files  (these 3 files) are added by your PR which should be removed:
Android/app/gradle/wrapper/gradle-wrapper.jar
Android/app/gradle/wrapper/gradle-wrapper.properties
Android/app/gradlew.bat

Rajnish23 commented:
Done, Please Review it.

On Fri, Oct 2, 2020 at 5:20 PM Swati Garg <notifications@github.com> wrote:

> The gradle files (these 3 files) are added by your PR which should be
> removed:
> Android/app/gradle/wrapper/gradle-wrapper.jar
> Android/app/gradle/wrapper/gradle-wrapper.properties
> Android/app/gradlew.bat
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#issuecomment-702761869>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHUZ7SMZGCLPLE6YMH2MUWLSIXOUHANCNFSM4SA45FIA>
> .
>

Swati4star commented on Android/app/gradle/wrapper/gradle-wrapper.properties:
```diff
@@ -0,0 +1,6 @@
+#Mon Apr 29 12:39:08 CEST 2019
```
??

Rajnish23 replied:
It's not added from my end.

Swati4star replied:
it is

Rajnish23 replied:
#Mon Apr 29 12:39:08 CEST 2019
distributionBase=GRADLE_USER_HOME
distributionPath=wrapper/dists
zipStoreBase=GRADLE_USER_HOME
zipStorePath=wrapper/dists
distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip

This is what I found in master branch. Anyway I have removed that line. Please review it.

Swati4star replied:
My comment wasn't about the line, it was about the file. I mentioned in my *previous comments* also to remove this file.

Anyways, I made a commit to remove that file.

Rajnish23 commented:
Thanks, Also please close the issue #822.

On Sat, 3 Oct 2020, 9:31 pm Swati Garg, <notifications@github.com> wrote:

> Merged #823 <https://github.com/project-travel-mate/Travel-Mate/pull/823>
> into master.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/project-travel-mate/Travel-Mate/pull/823#event-3836846114>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHUZ7SKV6HWKAD54ICIEY3DSI5UYVANCNFSM4SA45FIA>
> .
>
